packages/exports/src/gSuite/slides/deleteSlides.ts

```diff
@@ -2,8 +2,9 @@ import type { slides_v1 } from "@googleapis/slides";
 
 import type { Result } from "../../types";
 
-// #TODO Remove this string fallback and properly type the speaker notes tags
-export type SpeakerNotesTag = "cycle1" | "cycle2" | "cycle3" | string;
+export type SpeakerNotesTag =
```

Comment:
I'm not sure why this is called SpeakerNotesTag when it's actually a value indicating a slide that should be deleted

Reply:
I'm not sure either. How are speaker notes used?

Reply:
I think I was involved in this. The slides template has text in the speaker notes with the name of each slide. This code selectively deletes slides based on the text "tag" in the speaker notes

Reply:
Maybe we need a comment here. This type reflects the values available in the "speaker notes" of the template slides document

Reply:
Ah yes that rings a bell now!

Reply:
Added a comment